web/src/util/oktaAuth.js

```diff
@@ -14,14 +14,11 @@ const oktaAuth =
         clientId: OKTA_CLIENT_ID,
         redirectUri: OKTA_REDIRECT_URI,
         tokenManager: {
-          storage: 'cookie', // defaults to secure cookie
+          storage: 'localStorage', // defaults to secure cookie
```

Comment:
I've heard using localStorage is not the best security wise. We may want to investigate this more. Either way, I would update the comment.

Reply:
I would prefer to leave it as a cookie, but even with increasing the nginx header size it still didn't like how big our headers are with it. Also, it's less secure because it can be tampered with locally, but since it's being verified by Okta on the backend, that should be ok.